MovieChecker/MovieChecker/MovieList.h

```diff
@@ -0,0 +1,24 @@
+//
+//  MovieList.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/27/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol MovieListDelegate <NSObject>
+
+- (void)synchMoviesWithArray:(NSArray *)movies;
```

Comment:
Delegate methods should tell the receiver what the sender did (and, usually, should include a reference to the sender).  Perhaps something like `- (void)movieListController:(MovieListController *)movieListController didLoadMovies:(NSArray *)movies;`.